### ISSUE TITLE
My loss can not continue to drop

### ISSUE THREAD

**Issue by lzwhard** [pjreddie/darknet#316]
I used the voc data and yolo.cfg,
after 2100 iters, the avg loss is 19
after 9500 iters, the avg loss is 21.76
I don't know what's wrony, please help me out!

**Comment by AurusHuang** [comment]
My suggestions: 
1. Check your learning rate. Set a lower value. 
2. Use a different weights file. Perhaps this weights file doesn't fit your network.

**Comment by lzwhard** [comment]
Thank u, Aurus
I will try it

**Comment by SteveIb** [comment]
Hi, did it work with you?
my avg loss is going up and down between 2-6.
any tip?I lowered learning rate from learning_rate=0.0001
to learning_rate=0.00001

**Comment by barzan-hayati** [comment]
Hi @lzwhard  @SteveIb Have you resolved your problem in non decreasing average loss?

**Comment by mouad47** [comment]
Hi @barzan-hayati did you solve it !

**Comment by barzan-hayati** [comment]
> 
> 
> Hi @barzan-hayati did you solve it !

Set the learning rate in starting learning to 0.0001 or 0.0005 then increase it.
or use another weight file.
